pyproject.toml

```diff
@@ -96,7 +99,6 @@ profile = "black"
 
 # Configure pytest
 [tool.pytest.ini_options]
-addopts = "--cov=neurostatslib"   # Additional options to pass to pytest, enabling coverage for the 'neurostatslib' package
```

Comment:
why remove coverage?

I'm now realizing, have we enabled the codecov link to this package?

Reply:
we had the --cov in the call to pytest from tox. I added the converage in the pyproject.toml and removed it form the tox call to avoid redundancies.
codecov is not linked yet but I am working on it